package.json

```diff
@@ -9,10 +9,10 @@
     "webpack": "./node_modules/webpack/webpack.js"
   },
   "engines": {
-    "node": "^10.x"
+    "node": "^18.x"
   },
   "scripts": {
-    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn test && rm -rf client/dist/* && NODE_ENV=production webpack --mode production --bail --progress",
```

Comment:
`rm -rf client/dist/*` is added so that the entire dist directory is cleared out before building the resources. This means that, for example, if `moment-js` removes some file, we won't be retaining an old copy of it.
I expect this will be one of the more controversial changes and would welcome feedback on it.

Reply:
The "controversial" bit here is the use of `rm -rf` which won't work on windows environments using the default "command" terminal.  This isn't controversial in the slightest for me, Silverstripe has very poor native windows "command" support as far as I know and there's no chance of it would work.  I'd expect anyone using a windows machine to be using WSL or docker or similar